worlds/terraria/Rules.dsv

```diff
@@ -1,153 +1,200 @@
 Feeling Petty;                      Achievement;                                Cat | Dog;
 Dog;                                Npc;                                        Zoologist;
+Painter;                            Npc;                                        @npc(8);
 A Rather Blustery Day;              Achievement | Grindy;
 Enchanted Sword;
 Pretty in Pink;                     Achievement | Grindy;
 Marathon Medalist;                  Achievement | Grindy;
 Angler;                             Npc;
+Fisherman's Pocket Guide;           ;                                           Angler | Weather Radio;
+Weather Radio;                      ;                                           Angler | Sextant;
+Sextant;                            ;                                           Angler | Fisherman's Pocket Guide;
 Servant-in-Training;                Achievement | Fishing;                      Angler;
 \10 Fishing Quests;                 Achievement | Fishing;                      Angler;
 Trout Monkey;                       Achievement | Fishing;                      Angler;
 Glorious Golden Pole;               Achievement | Fishing;                      Angler;
 Fast and Fishious;                  Achievement | Fishing;                      Angler;
 Supreme Helper Minion!;             Achievement | Fishing;                      Angler;
 Water Walking Boots;
-Painter;                            Npc;                                        @npc(8);
+Aquatic Heart;                      Calamity;
 
 // Sky exploration
-Into Orbit;                         Achievement;
+Into Orbit;                         Achievement | Early;
 Mysterious Circuitry;               Calamity;
 Dubious Plating;                    Calamity;
 Charging Station;                   Calamity;
 Codebreaker Base;                   Calamity;
 Starfury;
+Celestial Magnet;
+Clumsy Slime;                       Npc;
 
 // Underground
-Watch Your Step!;                   Achievement;
-Throwing Lines;                     Achievement;
+Watch Your Step!;                   Achievement | Early;
+Throwing Lines;                     Achievement | Early;
 Torch God;                          Location | Item(Reward: Torch God's Favor);
-Vehicular Manslaughter;             Achievement;
-Hey! Listen!;                       Achievement;
+Vehicular Manslaughter;             Achievement | Early;
+Ancient Bone Dust;                  Calamity;
+Depth Meter;
+Compass;
+Hey! Listen!;                       Achievement | Early;
 I Am Loot!;                         Achievement;
+Magic Mirror;
 Heart Breaker;                      Achievement;
-Hold on Tight!;                     Achievement;
+Nurse;                              Npc;                                        Merchant;
+The Frequent Flyer;                 Not Calamity Getfixedboi | Achievement;     Nurse;
+Feast of Midas;                     Achievement;
```

Comment:
Shouldn't this require the Bug Net? Or are you just expecting the player to get it from Cans of Worms?